From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@alien8d__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@alien8d : Thanks for your contribution to the Azure documentation! The author, @joaoma, has been notified to review your proposed change.

joaoma commented:
Thank you! #sign-off

v-nagta committed:
USER STORY 1209402 SaaS App Tutorial: G Suite

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
Updated

v-nagta committed:
updated

v-nagta committed:
Video tutorial removed

carolinacmoravia committed:
M24798: Missing preposition in string 

Hello, @PatAltimore, 

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

carolinacmoravia committed:
M24914: Typo mistake: "JsonPropery"

Hello, @rnagpal, 

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

lachie83 committed:
Add GPU on AKS docs

v-viinde committed:
USER STORY 1228745 SaaS App Tutorial:OrgChart Now

v-viinde committed:
Updated

v-viinde committed:
Updated

lachie83 committed:
Grammar and formatting updates based on PR feedback

lachie83 committed:
General updates based on review
Added region availibilty notice
Add details on library and debug tools path mounting
Add Next step section

ktoliver committed:
edit pass: two Application Insights pricing articles

ktoliver committed:
more edits

ktoliver committed:
more edits

v-viinde committed:
Updated

ktoliver committed:
revisions per mrbullwinkle

ktoliver committed:
sync files

ktoliver committed:
sync files

v-viinde committed:
Updated

lachie83 committed:
Update container repo

tamram committed:
Update storage-introduction.md

tamram committed:
Update storage-introduction.md

Commit:
automation : refactor inventory article

Commit:
updates

Commit:
updates

Commit:
updates

douglaslMS committed:
Added 3 new, 3 updated articles plus screenshots for Fri release.

douglaslMS committed:
Fixed a broken link resulting from filename change.

douglaslMS committed:
Added 3 new articles to Table of Contents.

barbkess committed:
fix metadata

ktoliver committed:
revision per DaleKoetke

revision per DaleKoetke

ktoliver committed:
revision per DaleKoetke

revision per DaleKoetke

ktoliver committed:
edit

ktoliver committed:
edit

Commit:
Metadata updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aasmetadata1

douglaslMS committed:
Added Next Steps section to 3 new articles.

Commit:
Metadata updates

Commit:
fixing some stuff to boost acrolinx score

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aasmetadata1

douglaslMS committed:
Made fixes requested by PR reviewer.

Commit:
acrolinx

nitinme committed:
hadoop metadata updates

barbkess committed:
fix acrolinx

vladvino committed:
Added max policy size

tamram committed:
Update storage-introduction.md

BryanTrach-MSFT committed:
(forumsteam) fixes MicrosoftDocs/azure-docs#6983

Removed App Service Environment for Power Apps and replaced it with App Service Environment. There is no ASE for Power Apps.

barbkess committed:
acrolinx fixes

Commit:
Fix low Acrolinx score

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aasmetadata1

tfitzmac committed:
added ui elements to marketplace

barbkess committed:
acrolinx

GitHubber17 committed:
Merge pull request #35406 from v-nagta/gsuite

USER STORY 1209402 SaaS App Tutorial: G Suite

ktoliver committed:
Merge pull request #37224 from v-viinde/orgchartnow

USER STORY 1228745 SaaS App Tutorial:OrgChart Now

mattbriggs committed:
remove duplicate section

adegeo committed:
Added missing TOC entry

mattbriggs committed:
update PS

ktoliver committed:
Merge pull request #37628 from MicrosoftDocs/tamram-patch-1

Update storage-introduction.md

PRMerger16 committed:
Merge pull request #37711 from BryanTrach-MSFT/patch-16

(forumsteam) fixes MicrosoftDocs/azure-docs#6983

ktoliver committed:
Merge pull request #37684 from Minewiskan/aasmetadata1

Aasmetadata1

PRMerger17 committed:
Merge pull request #37717 from mattbriggs/mdb_20180412_ps

remove duplicate section

sethmanheim committed:
Freshness reviews

GitHubber17 committed:
Merge pull request #37714 from tfitzmac/0412uielements

added ui elements to marketplace

GitHubber17 committed:
Merge pull request #37716 from Thraka/sf1

Added missing TOC entry

jluk committed:
add GDPR export/delete instructions

barbkess committed:
fix warning

PRMerger15 committed:
Merge pull request #37706 from vladvino/patch-11

Added max policy size

kgremban committed:
Fix file extensions for issue 7076

PRMerger17 committed:
Merge pull request #37705 from nitinme/nitinme-metadata-updates

Hadoop metadata updates - batch 3

nitinme committed:
hadoop metadata changes

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-metadata-updates

ktoliver committed:
Merge pull request #37677 from barbkess/april2

fix metadata

AdamS-MSFT committed:
(ForumsTeam) fixed typo

fixed typo mnaage to manage

nitinme committed:
more metadata updates

Commit:
Merge pull request #37727 from kgremban/apr12-7076

Fix file extensions for issue 7076

jluk committed:
details on delete

dkamstra committed:
Aded content about levels and action specific behavior to address CRIs

dkamstra committed:
Replaced duplicate documentation with a link to the source article

PRMerger12 committed:
Merge pull request #37729 from Adam-Smith-MSFT/patch-1

(ForumsTeam) fixed typo

PRMerger13 committed:
Merge pull request #37723 from jluk/master

add GDPR export/delete instructions

Commit:
automation : clear up text about pre-req versions

ajlam committed:
Refactor c++ tutorial

ajlam committed:
update

anavinahar committed:
Updating for default == max

NICs, IPs, NSG rules

anavinahar committed:
Update azure-virtual-network-limits.md

ajlam committed:
fix password comment

anavinahar committed:
Update azure-virtual-network-limits.md

anavinahar committed:
Update azure-virtual-network-limits.md

GitHubber17 committed:
Merge pull request #37728 from nitinme/nitinme-metadata-updates

Hadoop metadata updates - Batch 4

Commit:
corrected links

GitHubber17 committed:
Merge pull request #37737 from ajlam/master

 Refactor c++ tutorial

PRMerger18 committed:
Merge pull request #37733 from georgewallace/aniupdate

automation : clear up text about pre-req versions

Commit:
corrections

lachie83 committed:
Clarify create and watch steps, add cleanup

Commit:
updates

Commit:
Merge pull request #37739 from anavinahar/patch-40

Updating for default == max

anavinahar committed:
Update azure-virtual-network-limits.md

PRMerger13 committed:
Merge pull request #37745 from anavinahar/patch-41

Update azure-virtual-network-limits.md

linda33wj committed:
Fix Azure SQL conn string & add note for file system config

linda33wj committed:
Add v1 v2 switcher for perf & fault tolerance topics

msjuergent committed:
add_us_east_2

msjuergent committed:
add_table

msjuergent committed:
add_spaces

Commit:
Merge pull request #37747 from linda33wj/master

Update ADF copy content

PRMerger12 committed:
Merge pull request #37754 from msjuergent/wadoc

add_us_east_2

Commit:
updates to remove ARM

genlin committed:
Update resource-manager-common-deployment-errors.md

PRMerger19 committed:
Merge pull request #37760 from genlin/patch-5

Update resource-manager-common-deployment-errors.md

carolinacmoravia committed:
Update active-directory-integrating-applications.md

Commit:
Merge pull request #37017 from carolinacmoravia/patch-660

M24798: Missing preposition in string

PRMerger19 committed:
Merge pull request #37023 from carolinacmoravia/patch-662

M24914: Typo mistake: "JsonPropery"

rjagiewich committed:
Merge pull request #37641 from georgewallace/inventoryupdate

automation : refactor inventory article

GitHubber17 committed:
Merge pull request #37697 from Blackmist/master

metadata

itechedit committed:
Merge pull request #37221 from lachie83/feat-add-aks-gpus

Add GPU on AKS docs

Commit:
Changed author

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Author-reassign-classic

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#7108

Updating mountpoint and doc author

dkamstra committed:
Addressed feedback from doc team

iainfoulds committed:
[VMSS] Fixes MicrosoftDocs/azure-docs#7069

tamram committed:
addressing feedback - broken link

Commit:
Merge pull request #37766 from iainfoulds/patch-2

[VM] Mount Azure file share with Linux

dkamstra committed:
Fixed table formatting problem by moving level content out of the table

Commit:
Edits.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Author-reassign-classic

PRMerger12 committed:
Merge pull request #37767 from iainfoulds/vmssautoscaletutorial

[VMSS] Scale-in actions when distributed across AZs

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Update analysis-services-gateway-install.md

GitHubber17 committed:
Merge pull request #37765 from jimdial/Author-reassign-classic

Author reassign classic

GitHubber17 committed:
Merge pull request #37731 from dkamstra/master

Aded content about levels and action specific behavior to address CRIs

PRMerger19 committed:
Merge pull request #37722 from sethmanheim/eh-fresh

Freshness reviews

v-alje committed:
Merge pull request #37773 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Update analysis-services-gateway-install.md

Ja-Dunn committed:
Merge pull request #37362 from ktoliver/1228224

edit pass: Two Application Insights pricing articles

PRMerger12 committed:
Merge pull request #37768 from tamram/tamram-0410

addressing feedback - broken link

PRMerger13 committed:
Merge pull request #37774 from Minewiskan/patch-1

Update analysis-services-gateway-install.md

v-shils committed:
Merge pull request #37667 from douglaslMS/adf-everest-3features

Added 3 new, 3 updated articles plus screenshots for Fri release.
